packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -106,6 +107,86 @@ item.addMethod('GET');   // GET /items/{item}
 item.addMethod('DELETE', new apigateway.HttpIntegration('http://amazon.com'));
 ```
 
+## AWS StepFunctions backed APIs
+
+You can use Amazon API Gateway with AWS Step Functions as the backend integration, specifically Synchronous Express Workflows.
+
+The `StepFunctionsRestApi` only supports integration with Synchronous Express state machine. The `StepFunctionsRestApi` construct makes this easy and also sets up input, output and error mapping. The `StepFunctionsRestApi` construct sets up the API Gateway REST API with an `ANY` HTTP method and sets up the api role with the required permission to invoke `StartSyncExecution` action on the AWS StepFunctions state machine. It sets up up a `prod` stage by default. This will enable you to invoke any of the API Gateway HTTP methods for that `prod` and get a response from the backend AWS Step Functions execution. Invoking either GET or POST in the example below will send the request to the state machine as a new execution. On success, an HTTP code '200' is returned with ONLY the execution output as the Response Body. If the state machine execution fails, an HTTP '500' error response is returned with the error and cause of the execution as the Response Body. If the request is invalid (ex. bad execution input) an HTTP '400' error is returned.
+
+As part of the API Gateway integration with Step Functions, it is possible to opt-in to include requestContext, headers, path, and querystring to the execution input. By default, these paramaters are not included in order to reduce payload size sent to Step Functions. 
+
+More details about AWS Step Functions payload limit can be found at https://docs.aws.amazon.com/step-functions/latest/dg/limits-overview.html#service-limits-task-executions.
+
+The following code defines a REST API that routes all requests to the specified AWS StepFunctions state machine:
+
+```ts
+const machineDefinition = new sfn.Pass(this, 'PassState', {
+
+})
+
+const stateMachine: sfn.IStateMachine = new sfn.StateMachine(this, 'StateMachine', {
+    definition: machineDefinition,
+    stateMachineType: sfn.StateMachineType.EXPRESS,
+});
+    
+new apigateway.StepFunctionsRestApi(this, 'StepFunctionsRestApi', { 
+  stateMachine: stateMachine,
+});
+```
+
+Here are a few examples:
+
+Example 1: POST with default configuration
+
+```json
+POST /
+{
+     "customerId": 1 
+}
+```
+
+Step Functions will wrap the request body inside a `body` key as it is possible to have other keys within the request (ex. requestContext, header, path, and/or querystring):
```

Comment:
It is accurate but I will remove to avoid confusion. The point here was to clarify that a request body will not be the whole execution input.

an API request with this body:

```json
{
    "customerId": 1
}
```

will be translated to:

```json
{
   "body": {
        "customerId": 1
   } 
}
```

as API Gateway can forward more fields (headers, path, requestContext) and they are not part of the request body.